scss/_layout.scss

```diff
@@ -6,4 +6,5 @@
 #{$ubtn-namespace}-block,
 #{$ubtn-namespace}-stacked {
   display: block;
-}
\ No newline at end of file
+  width: 100%;
```

Comment:
Can you also add 
box-sizing: border-box; 

This will allow the padding and borders to be included in the calculation for 100% preventing the width from exceeding 100%  and causing display issues. Thanks!